app/src/main/java/org/oppia/android/app/mydownloads/downloads/DownloadsTopicViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.android.app.mydownloads.downloads
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModel
+import org.oppia.android.app.home.RouteToTopicListener
+import org.oppia.android.app.model.TopicSummary
+import org.oppia.android.app.shim.IntentFactoryShim
+import org.oppia.android.util.parser.TopicHtmlParserEntityType
+
+/** [ViewModel] for title in [DownloadsFragment]. */
+class DownloadsTopicViewModel(
+  private val activity: AppCompatActivity,
```

Comment:
<img width="920" alt="Screenshot 2021-04-08 at 6 13 44 PM" src="https://user-images.githubusercontent.com/9396084/114028502-2c427000-9896-11eb-8892-7fedc3a8a53d.png">

Reply:
could you confirm this with the `CompletedStoryItemViewModel`?

Reply:
Yes, it is there too. That means it needs changes everywhere. You can skip this in this PR.

Reply:
Creating another issue.

Reply:
https://github.com/oppia/oppia-android/issues/3084